src/rebar_prv_install_deps.erl

```diff
@@ -85,7 +85,10 @@ do(State) ->
             no_cycle ->
                 case compile_order(Source, ProjectApps) of
                     {ok, ToCompile} ->
-                        {ok, rebar_state:deps_to_build(State1, ToCompile)};
+                        %% Deps may have plugins to install. Find and intall here.
+                        {ok, PluginProviders, State2} = rebar_plugins:install(State1),
+                        State3 = rebar_state:create_logic_providers(PluginProviders, State2),
+                        {ok, rebar_state:deps_to_build(State3, ToCompile)};
```

Comment:
No, nothing is compiled yet. It just means they don't work until after `install_deps` is complete.